lib/content-services/src/lib/category/services/category.service.spec.ts

```diff
@@ -177,4 +177,31 @@ describe('CategoryService', () => {
             expect(linkCategoriesSpy).toHaveBeenCalledOnceWith(fakeNodeId, fakeCategoriesLinkBodies);
         });
     }));
+
+    describe('areCategoriesEnabled', () => {
+        let getSpy: jasmine.Spy<(key: string, defaultValue?: boolean) => boolean>;
+
+        beforeEach(() => {
+            getSpy = spyOn(TestBed.inject(AppConfigService), 'get');
+        });
+
+        it('should call get on AppConfigService with correct parameters', () => {
+            categoryService.areCategoriesEnabled();
+            expect(getSpy).toHaveBeenCalledWith('plugins.categories', true);
+        });
+
+        it('should return true if get from AppConfigService returns true', () => {
+            getSpy.and.returnValue(true);
+
+            const enabledCategories = categoryService.areCategoriesEnabled();
```

Comment:
I think you don't need to create a extra variable here, you can just call `areCategoriesEnabled` in the expect block

Reply:
good catch, done